kinto/views/buckets.py

```diff
@@ -0,0 +1,39 @@
+from cliquet import resource
+
+from kinto.views import NameGenerator
+
+
+buckets_options = {
+    'collection_methods': ('GET',)
+}
+
+
+@resource.register(name="bucket", **buckets_options)
+class Bucket(resource.BaseResource):
+
+    def __init__(self, *args, **kwargs):
+        super(Bucket, self).__init__(*args, **kwargs)
```

Comment:
What do you have in mind ?

I would go for https://github.com/mozilla-services/cliquet/issues/32 instead of resource hooks :)